plugins/Compressor/CompressorControlDialog.cpp

```diff
@@ -0,0 +1,757 @@
+/*
+ * CompressorControlDialog.cpp
+ *
+ * Copyright (c) 2020 Lost Robot <r94231@gmail.com>
+ *
+ * This file is part of LMMS - https://lmms.io
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public
+ * License as published by the Free Software Foundation; either
+ * version 2 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this program (see COPYING); if not, write to the
+ * Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+ * Boston, MA 02110-1301 USA.
+ *
+ */
+
+#include <QLayout>
+
+#include "Compressor.h"
+#include "CompressorControlDialog.h"
```

Comment:
```suggestion
#include "CompressorControlDialog.h"

#include <QLayout>

#include "Compressor.h"
```
Related header file must be included first. See the [coding conventions](https://github.com/LMMS/lmms/wiki/Coding-Conventions#include-order).

Reply:
Oh, I fixed that on my local branch but completely forgot to push the changes.